[2025-06-02T15:48:32.184+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: google_scraper_pipeline.test_selenium_environment manual__2025-06-02T15:09:07.943661+00:00 [queued]>
[2025-06-02T15:48:32.187+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: google_scraper_pipeline.test_selenium_environment manual__2025-06-02T15:09:07.943661+00:00 [queued]>
[2025-06-02T15:48:32.187+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2025-06-02T15:48:32.192+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): test_selenium_environment> on 2025-06-02 15:09:07.943661+00:00
[2025-06-02T15:48:32.194+0000] {standard_task_runner.py:57} INFO - Started process 3574 to run task
[2025-06-02T15:48:32.196+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'google_scraper_pipeline', 'test_selenium_environment', 'manual__2025-06-02T15:09:07.943661+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/google_scraper_dag.py', '--cfg-path', '/tmp/tmp14_2nj42']
[2025-06-02T15:48:32.197+0000] {standard_task_runner.py:85} INFO - Job 18: Subtask test_selenium_environment
[2025-06-02T15:48:32.219+0000] {task_command.py:415} INFO - Running <TaskInstance: google_scraper_pipeline.test_selenium_environment manual__2025-06-02T15:09:07.943661+00:00 [running]> on host 1457ea4f74d2
[2025-06-02T15:48:32.250+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='google_scraper_pipeline' AIRFLOW_CTX_TASK_ID='test_selenium_environment' AIRFLOW_CTX_EXECUTION_DATE='2025-06-02T15:09:07.943661+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-02T15:09:07.943661+00:00'
[2025-06-02T15:48:32.250+0000] {logging_mixin.py:151} INFO - üìã A testar ambiente Selenium/Chrome
[2025-06-02T15:48:35.151+0000] {logging_mixin.py:151} INFO - ‚úÖ Depend√™ncias instaladas com sucesso
[2025-06-02T15:48:35.152+0000] {logging_mixin.py:151} INFO - üì¶ A executar comando: python3 /opt/***/dags/google_scraper/tests/test_selenium_environment.py
[2025-06-02T15:48:35.441+0000] {logging_mixin.py:151} INFO - C√≥digo de retorno: 1
[2025-06-02T15:48:35.441+0000] {logging_mixin.py:151} INFO - Resultado: 
[2025-06-02T15:48:35.442+0000] {logging_mixin.py:151} INFO - Erros: 2025-06-02 15:48:35,227 - INFO - Starting Selenium environment tests...
2025-06-02 15:48:35,429 - INFO - ‚úÖ Selenium and WebDriver Manager imports successful
2025-06-02 15:48:35,429 - ERROR - ‚ùå Chrome/Chromium not found at: /usr/bin/chromium
2025-06-02 15:48:35,429 - WARNING - Skipping browser test due to failed prerequisite checks
2025-06-02 15:48:35,429 - INFO - 
Test Summary:
2025-06-02 15:48:35,429 - INFO - Selenium Installation: ‚úÖ PASS
2025-06-02 15:48:35,429 - INFO - Chrome Environment: ‚ùå FAIL
2025-06-02 15:48:35,429 - INFO - Headless Browser: ‚ùå FAIL
2025-06-02 15:48:35,429 - ERROR - ‚ùå Some tests failed
[2025-06-02T15:48:35.442+0000] {logging_mixin.py:151} INFO - ‚ö†Ô∏è Comando falhou com c√≥digo de retorno 1
[2025-06-02T15:48:35.442+0000] {logging_mixin.py:151} INFO - ‚ö†Ô∏è Erro ao executar comando: Command '['python3', '/opt/***/dags/google_scraper/tests/test_selenium_environment.py']' returned non-zero exit status 1.
[2025-06-02T15:48:35.442+0000] {logging_mixin.py:151} INFO - Resultado: 
[2025-06-02T15:48:35.442+0000] {logging_mixin.py:151} INFO - Erro: 2025-06-02 15:48:35,227 - INFO - Starting Selenium environment tests...
2025-06-02 15:48:35,429 - INFO - ‚úÖ Selenium and WebDriver Manager imports successful
2025-06-02 15:48:35,429 - ERROR - ‚ùå Chrome/Chromium not found at: /usr/bin/chromium
2025-06-02 15:48:35,429 - WARNING - Skipping browser test due to failed prerequisite checks
2025-06-02 15:48:35,429 - INFO - 
Test Summary:
2025-06-02 15:48:35,429 - INFO - Selenium Installation: ‚úÖ PASS
2025-06-02 15:48:35,429 - INFO - Chrome Environment: ‚ùå FAIL
2025-06-02 15:48:35,429 - INFO - Headless Browser: ‚ùå FAIL
2025-06-02 15:48:35,429 - ERROR - ‚ùå Some tests failed
[2025-06-02T15:48:35.442+0000] {logging_mixin.py:151} INFO - ‚ö†Ô∏è Erro ao testar ambiente Selenium: Command '['python3', '/opt/***/dags/google_scraper/tests/test_selenium_environment.py']' returned non-zero exit status 1.
[2025-06-02T15:48:35.442+0000] {logging_mixin.py:151} INFO - ‚ö†Ô∏è O ambiente Selenium pode n√£o estar corretamente configurado!
[2025-06-02T15:48:35.442+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-06-02T15:48:35.449+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=google_scraper_pipeline, task_id=test_selenium_environment, execution_date=20250602T150907, start_date=20250602T154832, end_date=20250602T154835
[2025-06-02T15:48:35.497+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-06-02T15:48:35.506+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
